my_store/blog/dashboard/tables.py

```diff
@@ -0,0 +1,10 @@
+import django_tables2 as tables
```

Comment:
See how each module should order by this document https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/